internal/browser/browser_linux.go

```diff
@@ -0,0 +1,5 @@
+package browser
+
+func openArgv(url string) []string {
+	return []string{"xdg-open", url}
```

Comment:
i think the real upstream version is more complex now?
https://github.com/pkg/browser/blob/master/browser_linux.go#L9

Reply:
Ah, I was just grabbing it from `vendor`. I'll update.